substrate-sgx/externalities/src/lib.rs

```diff
@@ -129,6 +129,30 @@ pub fn with_externalities<F: FnOnce(&mut SgxExternalities) -> R, R>(f: F) -> Opt
 	ext::with(f)
 }
 
+/// Results concerning an operation to remove many keys.
```

Comment:
Do we need this?

Reply:
yes, else the worker doesn't compile. This is new code, that's copied from substrate